components/brave_vpn/resources/panel/constants/locale.ts

```diff
@@ -0,0 +1,6 @@
+const locale = {
+  connectedText: 'Connect',
```

Comment:
for follow-up, suffixing with "Text" doesn't seem neccessary since they will all be some kind of text. Might be good to instead differentiate with suffixes, e.g. "label" and "action" for different uses of the text and to aid translators.

Reply:
good point. I can do this on a follow up!